src/components/Button/Button.module.css

```diff
@@ -0,0 +1,35 @@
+.button {
+  display: flex;
```

Comment:
If we don't apply this style the button with the icon (or the one loading) is rendered badly because buttons have by default `display: inline-block`. However, I realized that we can remove `display: flex` and `align-items: center` from all buttons styles for size (`buttonSm`, `buttonMd`, `buttonLg`) because they are already inherited from here 💪🏼